WatsonDeveloperCloud/Dialog/Dialog.swift

```diff
@@ -19,49 +19,59 @@ import Alamofire
 import AlamofireObjectMapper
 import ObjectMapper
 
-/// The IBM Watson™ Dialog service provides a comprehensive, robust, platform for managing conversations between virtual agents and users through an application programming interface (API). These conversations are commonly referred to as dialogs.
+/**
+ The IBM Watson™ Dialog service provides a comprehensive, robust, platform for managing
+ conversations between virtual agents and users through an application programming
+ interface (API). These conversations are commonly referred to as dialogs.
+ */
 public class Dialog: WatsonService {
-    
+
     // The shared WatsonGateway singleton.
     let gateway = WatsonGateway.sharedInstance
-    
+
     // The authentication strategy to obtain authorization tokens.
-    var authStrategy: AuthenticationStrategy
-    
+    let authStrategy: AuthenticationStrategy
+
+    // TODO: comment this initializer
```

Comment:
This is an [existing issue](https://github.com/watson-developer-cloud/ios-sdk/issues/148) that will be tackled in a future [milestone](https://github.com/watson-developer-cloud/ios-sdk/milestones/Release%200.4.0:%20Code%20Quality).